---
title: "The statistics of identifying differentially expressed genes in Expresso and TM4: a comparison"
collection: publications
permalink: /publication/2006-04-statistics-of-identifying-significantly-expressed-genes
excerpt: ''
date: 2006-04-20
venue: 'BMC Bioinformatics'
paperurl: 'https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-215'
citation: 'Sioson, A.A., Mane, S.P., Li, P. et al. The statistics of identifying differentially expressed genes in Expresso and TM4: a comparison. BMC Bioinformatics 7, 215 (2006). https://doi.org/10.1186/1471-2105-7-215'
---
Analysis of DNA microarray data takes as input spot intensity measurements from scanner software and returns differential expression of genes between two conditions, together with a statistical significance assessment. This process typically consists of two steps: data normalization and identification of differentially expressed genes through statistical analysis. The Expresso microarray experiment management system implements these steps with a two-stage, log-linear ANOVA mixed model technique, tailored to individual experimental designs. The complement of tools in TM4, on the other hand, is based on a number of preset design choices that limit its flexibility. In the TM4 microarray analysis suite, normalization, filter, and analysis methods form an analysis pipeline. TM4 computes integrated intensity values (IIV) from the average intensities and spot pixel counts returned by the scanner software as input to its normalization steps. By contrast, Expresso can use either IIV data or median intensity values (MIV). Here, we compare Expresso and TM4 analysis of two experiments and assess the results against qRT-PCR data.
